bokeh/charts/builder/bar_builder.py

```diff
@@ -91,13 +114,15 @@ def get_data(self):
         # width should decrease proportionally to the value length.
         # 1./len(value) doesn't work well as the width needs to decrease a
         # little bit faster
-        width_cat = [min(0.2, (1./len(self._values))**1.1)] * len(self.cat)
+        width_cat = [min(0.2, (1. / len(self._values)) ** 1.1)] * len(self.cat)
         zero = np.zeros(len(self.cat))
         self._data = dict(
             cat=self.cat, width=width, width_cat=width_cat, zero=zero
         )
         # list to save all the groups available in the incomming input grouping
-        step = np.linspace(0, 1.0, len(self._values.keys()) + 1, endpoint=False)
+        step = np.linspace(0,
```

Comment:
In this case, I don't see value in readability just to avoid the 82 char... and also, in justified cases you can go up to 100, so, I would maintain this line as above...